.infrastructure/namespace.yml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name:
+    todoapp
```

Comment:
The 'name' field should be a single line key-value pair. It should be formatted as 'name: todoapp' on line 4 without indentation for the value.